src/infrastructure/content/ContentCommunicator.java

```diff
@@ -0,0 +1,24 @@
+package content;
+

```

Comment:
This content package is a sub package of infrastructure, so you need to write the entire path of the package. Here it will be infrastructure.content